lib/uniquable.rb

```diff
@@ -2,12 +2,14 @@ module Uniquable
   def self.unique_seasons_hash(data)
     game_by_season = data[:games]
     total_season = (game_by_season.uniq { |season| season[:season] }).map { |season| season[:season] }
-    total_count = Hash.new
```

Comment:
is there an advantage to using the Hash.new vs. {}??
speed?
performance?
convention?